README.md

```diff
@@ -186,6 +186,7 @@ Note the `match` field, which defines the scope of objects to which a given cons
    * `kinds` accepts a list of objects with `apiGroups` and `kinds` fields that list the groups/kinds of objects to which the constraint will apply. If multiple groups/kinds objects are specified, only one match is needed for the resource to be in scope.
    * `namespaces` is a list of namespace names. If defined, a constraint will only apply to resources in a listed namespace.
    * `labelSelector` is a standard Kubernetes label selector.
+   * `namespaceSelector` is a standard Kubernetes namespace selector.
```

Comment:
We should note that Namespaces need to be synced into GK for namespace selectors to work.